charts/kubewarden-controller/README.md

```diff
@@ -0,0 +1,82 @@
+Kubewarden is a Kubernetes Dynamic Admission Controller that uses policies written
+in WebAssembly.
+
+For more information refer to the [official Kubewarden website](https://kubewarden.io/).
+
+# kubewarden-controller
+
+`kubewarden-controller` is a Kubernetes controller that allows you to
+dynamically register Kubewarden admission policies.
+
+The `kubewarden-controller` will reconcile the admission policies you
+have registered against the Kubernetes webhooks of the cluster where
+it is deployed.
+
+## Installation
+
+The kubewarden-controller can be deployed using a helm chart:
+
+```shell
+$ helm repo add kubewarden https://charts.kubewarden.io
+$ helm install --create-namespace -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+```
+
+This will install kubewarden-controller on the Kubernetes cluster in the default
+configuration.
+
+The default configuration values should be good enough for the
+majority of deployments, all the options are documented
+[here](https://charts.kubewarden.io/#configuration).
+
+## Usage
+
+Once the kubewarden-controller is up and running, Kubewarden policies can be defined
+via the `ClusterAdmissionPolicy` resource.
+
+The documentation of this Custom Resource can be found
+[here](https://github.com/kubewarden/kubewarden-controller/blob/main/docs/crds/README.asciidoc)
+or on [docs.crds.dev](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller).
+
+**Note well:** `ClusterAdmissionPolicy` resources are cluster-wide.
+
+### Deploy your first admission policy
+
+The following snippet defines a Kubewarden Policy based on the
+[pod-privileged](https://github.com/kubewarden/pod-privileged-policy)
+policy:
+
+```yaml
+apiVersion: policies.kubewarden.io/v1alpha1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: privileged-pods
+spec:
+  module: registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.0
```

Comment:
Can you update that to be `v0.1.5`? I fear that is a really old version that would not work.

Also, I think you have copied that from somewhere else, can you update the original document too?

Reply:
Yes, I will come back to this. I have been fixing references to policies so we are on latest versions given some SDK changes, just to be on the safe side (waPC vs WASI, etc)